src/main/java/com/denys/toukdemo/dto/Seans.java

```diff
@@ -0,0 +1,44 @@
+package com.denys.toukdemo.dto;
+
+import java.util.Comparator;
+
+public class Seans {
+
+    private Integer id;
+    private Movie movie;
+    private Room room;
+    private String startDttm;
```

Comment:
co to jest Dttm?
wygląda jak data a jest Stringiem - czy to ok?

Reply:
> co to jest Dttm?
> wygląda jak data a jest Stringiem - czy to ok?

jak odpowiedziałem w komentarzu w entity gdzie to pole jest timestamp zrobiłem stringem dla prezentacji pola w czytelnym formacie